### PR TITLE
Update script editor panel tooltips

### LINKED ISSUE #3802: Missing tooltips on Script Editor panel
**Home Assistant release with the issue:**
0.99.2

**Description of problem:**
Another smaller issue for Hacktoberfest. Script Editor panel could use some improvements to tooltips - some are missing, some can be a little better.

**Add new tooltips**
* The pencil icon has no tooltip when hovered, it should say "Edit Script"
* The play icon for running scripts has no tooltip when hovered, it should say "Run Script"

**Tweak for better clarity**
* The tooltip when hovering over the plus button is "Add script" but I think that "Create New Script" is a bit more clear
* Similarly, the tooltip when hovering over the floating action button for SAVE is just "Save" when it should say "Save Script"

### PR DESCRIPTION
**Pull request by da-snap**
## Proposed change

Missing tooltips on Script Editor panel #3802

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #3802
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @da-snap, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;frontend#7204).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by iantrich — APPROVED** [review]

**Comment by iantrich** [comment]
🎉🎉🎉

File: src/translations/en.json [hunk 1]
```diff
@@ -1274,9 +1274,10 @@
             "introduction": "The script editor allows you to create and edit scripts. Please follow the link below to read the instructions to make sure that you have configured Home Assistant correctly.",
             "learn_more": "Learn more about scripts",
             "no_scripts": "We couldn’t find any editable scripts",
-            "add_script": "Add script",
```
Comment (bramkragten) [inline src/translations/en.json:1277]: Please don't rename the key, but just change the translation. Renaming the key means all translations are lost

Comment (da-snap) [reply src/translations/en.json:1277]: I changed it back. Is it displayed in other languages if only the translation has changed?

**Pull request merged**